src/parser/parser_driver.cpp

```diff
@@ -1278,6 +1278,27 @@ void ParserDriver::defineGrammar()
 			output->setTokenStream(currentTokenStream());
 			return output;
 		})
+		.production("primary_expression", "PERCENT", "OF", "string_set", [&](auto&& args) -> Value {
+			auto for_expr = std::move(args[0].getExpression());
+			TokenIt percent = args[1].getTokenIt();
+			size_t value = 0;
```

Comment:
Any specific reason for `size_t` which is usually used when we are dealing with memory sizes and not something like `uint8_t` since we are storing just values between 1 and 100?

Reply:
I see, thank you, I fixed it :+1: